jobrunner/config.py

```diff
@@ -247,9 +247,11 @@ def parse_job_resource_weights(config_file):
 # docker specific exit codes we understand
 DOCKER_EXIT_CODES = {
     # 137 = 128+9, which means was killed by signal 9, SIGKILL
+    # This could be killed externally by an admin, or terminated through the
+    # cancellation process.
     # Note: this can also mean killed by OOM killer, but that's explicitly
     # handled already.
-    137: "Killed by an OpenSAFELY admin",
+    137: "Job killed",
```

Comment:
see other comment thread - I thought if OOMKilled is unreliable we may be returning this for OOM jobs, so we should be more vague (rather than confidently saying something inaccurate)

Reply:
Yeah, I think I may have got confused.  This is only used if a) the job wasn't explicitly cancelled by a user or an admin (via cli.kill_Job) and b) OOMKilled is false.

But maybe "Job killed" is a bit too little information. Perhaps "Job killed by admin or memory limits"?

Reply:
🤝